datetimeparser/Evaluator.py

```diff
@@ -1,14 +1,54 @@
-from datetime import datetime
+from datetime import datetime, timedelta
 
 from .baseclasses import *
 
 
+def eastern_calc(year: int) -> str:
+    A = year % 19
+    K = year // 100
+    M = 15 + (3*K+3) // 4 - (8*K+13) // 25
+    D = (19*A+M) % 30
+    S = 2 - (3*K+3) // 4
+    R = D // 29 + (D//28-D//29) * (A//11)
+    OG = 21 + D + R
+    SZ = 7 - (year+year//4+S) % 7
+    OE = 7 - (OG-SZ) % 7
+    OS = OG + OE
+
+    if OS > 32:
+        return f"{year}-04-{OS-31} 0:00:00"
+    else:
+        return f"{year}-03-{OS} 0:00:00"
+
+curent_date = datetime.strptime(datetime.strftime(datetime.today(), "%Y-%m-%d"), "%Y-%m-%d")
+curent_datetime = datetime.strptime(datetime.strftime(datetime.today(), "%Y-%m-%d %H:%M:%S"), "%Y-%m-%d %H:%M:%S")
+
 class Evaluator:
+    EVENTS = {
```

Comment:
Change Dict names to Constant Enum Values so we have one single value that we can change all the time instead of multiple values flying around